server/create.go

```diff
@@ -330,6 +330,31 @@ func createModel(r api.CreateRequest, name model.Name, baseLayers []*layerGGML,
 		layers = append(layers, layer.Layer)
 	}
 
+	if len(r.ControlVectors) > 0 {
+		for _, cv_m := range r.ControlVectors {
```

Comment:
As of right now:
- We can support multiple control vectors
- The order of control vectors matters (as it correlates to the order of the control_strength parameter)

So rather than taking a `map[filename]digest`, it is a `[]map[filename]digest]` where each entry in the slice is one control vector, and each control vector is defined in the same way as the other files in the api